autobuild/examples/alldaemons.yaml

```diff
@@ -18,36 +18,41 @@
       image: blocknetdx/servicenode:latest # DO NOT DELETE
       config_mount_dir: /snode # DO NOT DELETE
       data_mount_dir: /snode # DO NOT DELETE
+    # AVALANCHE INDEXER - DELETE IF NOT NEEDED  
     - name: AVAX # Requires 200 GB of disk, 16 GB RAM & 6 vCPUs (23 Nov, 2021)
       image: avaplatform/avalanchego:v1.7.2
       data_mount_dir: /snode
-    - name: BTC # Requires 415.5 GB of disk (7 June, 2021)
+    # THE STANDARD XLITE WALLET SET - DELETE ANY YOU DON'T NEED  
+    - name: BTC # Requires 451 GB of disk (15 December, 2021)
       image: blocknetdx/bitcoin:v0.20.0
       config_mount_dir: /snode
       data_mount_dir: /snode
-    - name: LTC # Requires 66 GB of disk (4 Nov, 2021)
-      image: blocknetdx/litecoin:v0.18.1
+    - name: DASH # Requires 32 GB of disk (15 December, 2021)
+      image: blocknetdx/dash:v0.17.0.3
       config_mount_dir: /snode
       data_mount_dir: /snode
-    - name: DGB # Requires 27 GB of disk (4 Nov, 2021), 4-5 GB RAM/Swap
+    - name: DGB # Requires 27 GB of disk (15 December, 2021), 4-5 GB RAM/Swap
       image: blocknetdx/digibyte:v7.17.2
       config_mount_dir: /snode
       data_mount_dir: /snode
-    - name: SYS # Requires 3 GB of disk (4 Nov, 2021)
-      image: blocknetdx/syscoin4:v4.3.0
+    - name: LTC # Requires 71 GB of disk (15 December, 2021)
+      image: blocknetdx/litecoin:v0.18.1
       config_mount_dir: /snode
       data_mount_dir: /snode
-    - name: PIVX # Requires 18 GB of disk (4 Nov, 2021)
+    - name: PIVX # Requires 19 GB of disk (15 December, 2021)
       image: blocknetdx/pivx:v5.3.2
       config_mount_dir: /snode
       data_mount_dir: /snode
+    - name: SYS # Requires 4 GB of disk (15 December, 2021)
+      image: blocknetdx/syscoin4:v4.3.0
+      config_mount_dir: /snode
+      data_mount_dir: /snode
+    # OTHERS - DELETE ANY YOU DON'T NEED  
```

Comment:
Can we change this comment to:
\# OTHERS - DELETE ANY YOU DON'T WANT TO SUPPORT
?
Also, it occurs to me that the "STANDARD XLITE WALLET SET" will *hopefully* be expanding soon. For example, there was talk recently of including LBC in XLite. @rburget has already added support for LBC to XLite, actually. Now we just need to add the Blocknet snodes to the XLite backend through xrouter-js and we'll have LBC in XLite, and potentially a lot more coins added very quickly. As such, I'm wondering if we want to even define a "STANDARD XLITE WALLET SET," and to distinguish that set from "OTHERS," given our common hope that the set supported by XLite will soon be expanding rapidly...

Reply:
Done.

Yeah, soon (TM) the set will be bigger but for now it's not and it makes it easy for people to see which are supported by XLite and which are not. Eventually it will be possible to deploy a containerised trading setup with BlockDX and the sizes on disk and inclusion (or not) in the XLite set will be useful for those users too.